entrypoint.sh

```diff
@@ -15,10 +15,15 @@ fi
 # On PROD, we always collect statics
 if [ "$ENV" == "PROD" ]; then
     python3 manage.py collectstatic --no-input
+    python3 manage.py update_integrator_permissions
+    python3 manage.py compilemessages -l fr
+fi
+
+if [ "$ENV" == "DEV" ]; then
```

Comment:
Can you do an "else" or it's not possible?

Reply:
Autre fix possible :

if [ "$ENV"]; then
    python3 manage.py update_integrator_permissions
    python3 manage.py compilemessages -l fr
fi


Car le code ne fonctionne pas sans donnée en partant de 0 avec le code actuel.